### ISSUE TITLE
Traffic not goes through routes at Fedora 33

### ISSUE THREAD

**Issue by rma945** [aws/amazon-vpc-cni-k8s#1614]
**What happened**:
I have built a new AMI based on community Fedora 33 1.2 AMI with bootstrap scripts from this repository - https://github.com/awslabs/amazon-eks-ami and everything works fine, except the aws-vpc-cni. I have checked the nodes and found that pods are successfully created, the elastic interfaces are allocated and the routing tables are created successfully, but the pods can't ping or connect through a TCP to any external or local IP. But when I change the CNI plugin to calico - pods are able to reach any IP. The second worker node, based at AWS EKS AMI - works fine, the problem only with a Fedora-based AMI worker. Also, I was tried to switch the container runtime from docker to pure contained, but that not helps, the aws-vpc-cni still not works.

**What you expected to happen**:
Pods should be able to connect to local and internal services

**How to reproduce it (as minimally and precisely as possible)**:
Get the Fedora 33 1.2 AMI, join it into the EKS cluster and add aws-vpc addon

**Attached logs***
[Uploading eks_i-0cca70aab4bdd2bc1_2021-09-13_0719-UTC_0.6.2.tar.gz…]()

**Anything else we need to know?**:
**Environment**:
- Kubernetes version: 1.21
- CNI Version: v1.9.0-eksbuild.1
- OS: Fedora 33 (Cloud Edition)
- Kernel:  5.8.15-301.fc33.x86_64

**Comment by jayanthvn** [comment]
Hi @rma945 

Can we capture tcpdump on any of the nodes to verify if there is any issue with the ip tables or rules? Say when you ping from Pod-a to Pod-b

install tcpdump on the node
start capturing traffic on eth0 of node (assuming eth0 is the ENI for Pods)
`tcpdump -i eth0 -w node_a_eth0.pcap`

You can attach the pcap to the issue.

Also the logs attached above doesn't seem to download so can you reattach it?

**Comment by rma945** [comment]
There is it
[node_a_eth0.zip](https://github.com/aws/amazon-vpc-cni-k8s/files/7160281/node_a_eth0.zip)

**Comment by jayanthvn** [comment]
Can you confirm the source and destination Pod IPs and also did you ping or curl?

**Comment by rma945** [comment]
Yes, any address except 127.0.01 can't be accessed

**Comment by RomanCherednikovAZ** [comment]
yes, I have already tried to disable the NetworkManager routing rules, as suggested at this [issue](https://github.com/aws/amazon-vpc-cni-k8s/issues/1514) but this not help. Also, in my case - looks like that the routes are added properly, but the routing is blocked by some reason

```bash
[root@ip-172-24-67-130 ~]# ip rule list
0:      from all lookup local
512:    from all to 172.24.67.25 lookup main  <- pod cni ip
1024:   from all fwmark 0x80/0x80 lookup main
32766:  from all lookup main
32767:  from all lookup default
```

**Comment by jayanthvn** [comment]
Thanks for checking @RomanCherednikovAZ . Sorry I meant in the pcap file attached, can you please confirm the source and destination IPs ? If you have some bandwidth can you also please capture tcp dump on the destination side? Basically if we have the tcp dump on the source and destination nodes then we can co-relate where the drop is happening. 

Also can you please check if the security groups are configured properly?

**Comment by rma945** [comment]
Sorry for the confusion you, but rma945 and RomanCherednikovAZ = the same person. Yeah, I have checked that the CNI binaries were successfully added at the node, and the Network state for the node was changed to Ready. So the CNI itself - works fine, but looks like there were some issue with routing at the node. Anyway, at this moment we were anyway migrate our worker nodes back to the AWS EKS AMI.
